src/sidebar/zeplinComponent/tree/zeplinComponentTreeUtil.ts

```diff
@@ -0,0 +1,30 @@
+import TreeItem from "../../../common/vscode/tree/TreeItem";
+import ZeplinComponentSection from "../../../common/domain/zeplinComponent/model/ZeplinComponentSection";
+import ResponseZeplinComponent from "../../../common/domain/zeplinComponent/model/ResponseZeplinComponent";
+import ZeplinComponentTreeItem from "./ZeplinComponentTreeItem";
+import ZeplinComponentSectionTreeItem from "./ZeplinComponentSectionTreeItem";
+import localization from "../../../localization";
+
+function createList(components: ResponseZeplinComponent[], sections: ZeplinComponentSection[]): TreeItem[] {
+    const filledSections = sections?.filter(isSectionFilled) ?? [];
+    if (components.length || filledSections.length) {
+        return [
+            ...components.map(component => new ZeplinComponentTreeItem(component)),
+            ...filledSections.map(ZeplinComponentSectionTreeItem.fromSection)
+        ];
+    } else {
```

Comment:
No need else here.

Reply:
Yeah it is not needed, but I think it is more readable like this.

if this then that, else something else.

I generally do not use else in flow files in this project, because I consider them assertions.
for example:
```
is this not ok? then stop
is that not ok? then stop
do sth.
```

But what I did in this method is not an assertion